src/core1_task.rs

```diff
@@ -759,10 +759,31 @@ pub fn core_1_task(
                 // Finalise on a different frame period to writing out the prev/last frame,
                 // to give more breathing room.
                 if let Some(cptv_stream) = &mut cptv_stream {
-                    error!("Ending current recording");
                     let cptv_start_block_index = cptv_stream.starting_block_index as isize;
-                    cptv_stream.finalise(&mut flash_storage);
-                    let cptv_end_block_index = flash_storage.last_used_block_index.unwrap();
+
+                    if !making_status_recording
+                        && motion_detection.as_ref().unwrap().was_false_positive()
+                    // && cptv_stream.num_frames <= 100
+                    {
+                        info!(
+                            "Discarding as a false-positive {}:{} ",
+                            cptv_start_block_index, flash_storage.last_used_block_index
+                        );
+                        let _ = flash_storage.erase_last_file();
+                        event_logger.log_event(
+                            LoggerEvent::new(
+                                LoggerEventKind::WouldDiscardAsFalsePositive,
+                                synced_date_time.get_timestamp_micros(&timer),
+                            ),
+                            &mut flash_storage,
+                        );
+                    } else {
+                        cptv_stream.finalise(&mut flash_storage);
```

Comment:
It looks like now if we *are* making a status recording, it wouldn't get finalised?